package.json

```diff
@@ -1,13 +1,82 @@
 {
-  "name": "vibc-core-smart-contracts",
-  "version": "1.0.0",
-  "main": "index.js",
-  "repository": "https://github.com/open-ibc/vibc-core-smart-contracts",
+  "name": "@open-ibc/vibc-core-smart-contracts",
+  "version": "0.1.3",
```

Comment:
this should match with the current version.

Reply:
By current version, I'm assuming you mean the version that the main branch of this repo is tagged with right? 

if so, I'm leaving the version to a lower version for now so that we can set it to the current version once we finalize this PR (since there might be a couple more iterations before we finalize)

Reply:
npm is pretty strict about allowing to re-publish versions so I think we should only publish to v2.1.0 once we're sure it's finalized

Reply:
When this PR is merged we should bump the v2.1.0 to v2.2 or v3. Let's keep the one version for tags and "node version" to keep things simple